view.html.erb

```diff
@@ -1,6 +1,29 @@
-<form action="/rnode/<%= host %>/<%= port %>/login" method="post" target="_blank">
-  <input type="hidden" name="password" value="<%= password %>">
+<%
```

Comment:
This page is nearly copied verbatim from the Stanford batch connect app. As a result, the license of this app should change to match the Standford license, and in particular, some comment should be added to this file explaining that this is the portion copied from the Standford app.

Reply:
Added GNU license to `~/LICENSE`